service/src/main/resources/application.yaml

```diff
@@ -87,7 +87,7 @@ docmosis:
     key: ACCESS_KEY
 
 document_management:
-  url: http://localhost:4506
+  url: http://localhost:3453
```

Comment:
why?

Reply:
@akomisarek @joez-f

Reply:
@joez-f can we revert this? Master still reads 4506

Reply:
adam is happy with this change - proxy magic that means we can view documents without having to change the URL from 4506 to 3453 every time